### PR TITLE
Refactored HN Message parser

### PR DESCRIPTION
**Pull request by TiagoGraf89**

### CODE REVIEW

File: Apps/Medication/src/Parsers/TRPMessageParser.cs [hunk 1]
```diff
@@ -27,96 +27,26 @@ namespace HealthGateway.Medication.Parsers
...
-            // ZCB|BCXX000034|190819|248876
-            // \r
-            // ZCC||||||||||0009735353315|
-            // \r
-            // \r
-            CultureInfo culture = CultureInfo.CreateSpecificCulture("en-CA");
-            culture.DateTimeFormat.DateSeparator = "/";
-
-            HL7Encoding encoding = new HL7Encoding();
+            id = id.PadLeft(13, '0');
```
Comment (FuriousLlama) [inline Apps/Medication/src/Parsers/TRPMessageParser.cs]: Can you add a small blur about why we need to do this. My bad :)

**Comment by sslaws** [comment]
SonarQube Report: https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_137

**Comment by healthgatewaybc** [comment]
SonarQube Report: https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=commonlibrary_pr_137

**Comment by sslaws** [comment]
SonarQube Report: https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_137

**Pull request merged**